Dockerfile.ubi

```diff
@@ -0,0 +1,16 @@
+# Build the manager binary
```

Comment:
Is this file part of OADP-662?

Reply:
it is not

Reply:
So why include it here?

Reply:
>new Velero deps

Sounds more like a change to an existing dockerfile?

Including a new ubi image seems a scope for a separate PR 🤷

Reply:
At least the ubi dockerfile would be in addition to, not instead of, changing current dockerfile to work.

Reply:
This PR needed Velero deps to be added to go.mod and the build was failing, we needed to update the builder, at the same time we wanted add the ubi dockerfile. Hence the change. Maybe @eemcmullan fix the existing non-ubi dockerfile as well along with the ubi dockerfile addition ?

Reply:
@shubham-pampattiwar I can do that. I would like to keep the dockerfile changes in this PR though as they are needed to build these changes, as well as avoid rebasing.

Reply:
+1, I am good with that.

Reply:
+1